packages/json-machete/src/types.ts

```diff
@@ -1 +1 @@
-export { JSONSchema, JSONSchemaObject } from '@json-schema-tools/meta-schema';
+export type { JSONSchema, JSONSchemaObject } from '@json-schema-tools/meta-schema';
```

Comment:
Do we need a changeset for this?

Reply:
Nope, the dist files are exactly the same. This change affects testing and dev env only.